src/AzureAppConfigurationImpl.ts

```diff
@@ -371,12 +465,7 @@ export class AzureAppConfigurationImpl implements AzureAppConfiguration {
     }
 }
 
-function getValidSelectors(selectors?: SettingSelector[]) {
-    if (!selectors || selectors.length === 0) {
-        // Default selector: key: *, label: \0
-        return [{ keyFilter: KeyFilter.Any, labelFilter: LabelFilter.Null }];
-    }
-
+function validateSelectors(selectors: SettingSelector[]) {

```

Comment:
I liked the name `getValidSelectors` for this method because we are returning selectors, not just validating them.

Reply:
name updated with return type declaration added. 1182638